nion/instrumentation/DriftTracker.py

```diff
@@ -78,6 +81,114 @@ def __drift_changed(self, offset_nm: Geometry.FloatSize, elapsed_time: float) ->
         self.__event_loop.call_soon_threadsafe(functools.partial(self.__update_drift_log_data_item, delta_nm_data))
 
 
+@dataclasses.dataclass
+class DriftResult:
+    """Result of drift calculation returned by DriftCalculator.calculate
+
+    It can interpreted as: Between 'time_window_start' and 'time_window_end' the average drift rate was estimated to
+    be 'drift_rate'.
+
+    'drift_rate' should be in units of nm / s.
+    """
+
+    drift_rate: Geometry.FloatSize
```

Comment:
And ``drift = drift_rate * (end - start)`` so we could also simply add a method to get that. The reason I put in ``drift_rate`` as the "native" quantity is that it is more universal with respect to different calculation methods. Since the cross-correlation algorithm is seperated out now, we don't know between which images (i.e. first and last or second to last and last, etc) drift was calculated. A drift rate is independent from that, so you don't need additional information in order to interpret the result.